armi/physics/neutronics/globalFlux/tests/test_globalFluxInterface.py

```diff
@@ -0,0 +1,150 @@
+"""
+Tests for generic global flux interface.
+"""
+import unittest
+
+import numpy
+
+from armi import settings
+from armi.reactor import reactors
+
+from armi.physics.neutronics.globalFlux import globalFluxInterface as gi
+from armi.reactor.tests import test_reactors
+from armi.tests import ISOAA_PATH
+from armi.nuclearDataIO import isotxs
+
+
+class MockParams:
+    pass
+
+
+class MockCore:
+    def __init__(self):
+        self.geomType = "spiral"
```

Comment:
Either change or add comment explaining that this is not currently a supported geom type.
I understand we want to keep things light, but clarity is also important.